tests/test_helper.py

```diff
@@ -17,3 +20,119 @@
     FONT = (
         "Liberation-Mono"  # This is available in the fonts-liberation package on Linux.
     )
+
+
+def bitmap_to_frame_list(bitmap_frames, color_dict=None):
+    """
+    Turns a list of strings into a list of numpy arrays, each letter representing a (r, g, b) tuple.
+    Example input (2 frames, 5x3 pixel size):
+    [["RRRRR",
+      "RRBRR",
+      "RRBRR"],
+     ["RGGGR",
+      "RGGGR",
+      "RGGGR"]]
+    """
+    # "O" represents black
+    # "X", "Y", "Z" are arbitrary colours
+    if color_dict is None:
+        color_dict = {
+            "R": (255, 0, 0),
+            "G": (0, 255, 0),
+            "B": (0, 0, 255),
+            "O": (0, 0, 0),
+            "W": (255, 255, 255),
+            "X": (89, 225, 62),
+            "Y": (113, 157, 108),
+            "Z": (215, 182, 143),
+        }
+
+    output_array = []
+    for input_frame in bitmap_frames:
+        output_frame = []
+        for row in input_frame:
+            output_frame.append([color_dict[color] for color in row])
+        output_array.append(np.array(output_frame))
+
+    return output_array
+
+
+def bitmap_to_clip(bitmap_frames, color_dict=None):
+    """
+    Takes input of the same form as `bitmap_to_frame_list`, but then creates a VideoClip with the
+    result. The VideoClip has length in seconds equal to the number of frames, and fps is 1.
+    :param bitmap_frames:
+    :return:
+    """
+    frame_list = bitmap_to_frame_list(bitmap_frames, color_dict=color_dict)
+    make_frame = lambda t: frame_list[t]
```

Comment:
as a rule of thumb, don't assign lambdas to variables names. because the it spirit is to be "anonymous".  Pass it directly to `make_frame` argument or, better, define it properly with `def`.  

btw, this would be warning in a pycodestyle / flake8 linting. shouldn't we use it in our CI ?

Reply:
Will do, thanks for catching that one. They are caught by flake8 in the travis build, but not alerted anywhere because there are 500 or so warnings. (https://travis-ci.org/github/Zulko/moviepy/jobs/669677296#L381). Perhaps something to look at and decide which warnings to alert, and which to ignore?